docs/adr/0002-javascript-client-side.md

```diff
@@ -0,0 +1,25 @@
+# 1. Javascript for Client Side
+
+Date: 2019-08-05
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decision to use Javascript on the client-side code of Libero Reviewer.
```

Comment:
not much in terms of context, most of the `Discussion` (UX and chat/notification requirements) should go in here

Reply:
Let me make a second attempt...